### ISSUE TITLE
Don’t you feel ashamed?

### ISSUE THREAD

**Issue by finderound** [FairyMeng/TrustSR#1]
You are a mediocre tailor, foolish but not clever, copying and pasting everywhere, and in the end, piecing together a pile of bull-shit. Don’t you feel any guilt? Now even animals can publish papers?

**Comment by FairyMeng** [comment]
I can understand the poorly of life that makes you say such word, but keyboard warriors really don't know how to write papers.

**Comment by finderound** [comment]
You're right. I'm not good at producing garbage and making it public. A master tailor wouldn't piece shit together , but you're the exactaly opposite. You're not a skilled copier, but shamelessness is your best paper.
